mirascope/core/base/_utils/_convert_messages_to_message_params.py

```diff
@@ -49,12 +63,32 @@ def _convert_message_sequence_part_to_content_part(
             image=image,
             detail=None,
         )
+    elif has_pydub_module and isinstance(message_sequence_part, AudioSegment):
+        return AudioPart(
+            type="audio",
+            media_type="audio/wav",
+            audio=cast(
+                FileIO,
+                message_sequence_part.set_frame_rate(FRAME_RATE)
+                .set_channels(CHANNELS)
+                .set_sample_width(SAMPLE_WIDTH)
+                .export(format="wav"),
+            ).read(),
+        )
+    elif isinstance(message_sequence_part, Wave_read):
+        return AudioPart(
+            type="audio",
+            media_type="audio/wav",
+            audio=message_sequence_part.readframes(-1),
+        )
     else:
         raise ValueError(f"Invalid message sequence type: {message_sequence_part}")
 
 
 def convert_message_content_to_message_param_content(
-    message_sequence: Sequence[str | Image.Image | TextPart | ImagePart | AudioPart],
+    message_sequence: Sequence[
+        str | Image.Image | TextPart | ImagePart | AudioPart | AudioSegment | Wave_read
```

Comment:
nit: here and below, can we reorder so `Image.Image` is after `ImagePart` so the different input types are colocated better?